packages/app/src/components/uiBlocks/Dropdown.vue

```diff
@@ -45,11 +45,12 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, ref } from 'vue';
+import { defineComponent, PropType, ref } from 'vue';
```

Comment:
PropType never used? And also, what is it?

Reply:
I needed that to properly type a prop we have in that component. For more info you can have a look on [this comment](https://stackoverflow.com/a/65642306/8461267).